style.css

```diff
@@ -261,3 +261,111 @@ img {
 .border-lines {
   border-bottom: 0.5px solid #7a869a;
 }
+
+.contact-page {
+  display: flex;
+  flex-direction: column;
+  background-image: url('./img/contact--mobile.png');
+  background-size: 100%;
+  width: 100%;
+  height: 100%;
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  border-radius: 65px 0 0 0;
+}
+
+.contactme {
+  /* Primary Text */
+
+  width: 90%;
+  font-weight: 700;
+  font-size: 40px;
+  margin-bottom: -1%;
+  text-align: center;
+
+  /* Header / 2
+font-family: 'Poppins';
+font-style: normal;
+font-weight: 700;
+font-size: 40px;
+line-height: 52px;
+
+/* Neutrals/N0 */
```

Comment:
- [ ] [Optional] I would suggest you remove all commented codes from your work, as leaving commented codes in your code makes it unclean and messy. 😄 👏 ♻️

Reply:
@brainconnect93  Thank you very much, sir, I will do that.